asdf_astropy/tags/transform/basic.py

```diff
@@ -0,0 +1,216 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from asdf.versioning import AsdfVersion
+import asdf
+
+from astropy.modeling import models
+from astropy.modeling.core import CompoundModel, Model
+
+from . import _parameter_to_value
+
+
+__all__ = ['TransformType', 'IdentityType', 'ConstantType']
+
+
+class TransformType(asdf.AsdfConverter):
+    tags = {"http://asdf-format.org/schemas/transform/transforms-1.0.0"}
+    types = {Model}
+    version = '2.0.0'
```

Comment:
The `version` and `requires` attributes are no longer recognized by the framework